Source/Particles/LaserParticleContainer.cpp

```diff
@@ -585,7 +585,7 @@ LaserParticleContainer::Evolve (int lev,
             plane_Yp.resize(np);
             amplitude_E.resize(np);
 
-            if (rho && ! skip_deposition) {
+            if (rho && ! skip_deposition && ! do_not_deposit) {
```

Comment:
Unrelated to the PR that fixes it: Wow, this is some messy logic already in place from us.